### DISCUSSION TITLE
python version

### DISCUSSION THREAD

**Discussion by iblissstudent** [PlummersSoftwareLLC/Primes#942] [Ideas]
i made this in python:

```python
def IsPrime(Number):
	if str(Number).endswith('.0') or str(Number).endswith('.0') or Number % 2 == 0:
		return False

	passes = 0
	for num in range(3, int(float(Number)**0.5 + 1)):
		if Number % num == 0:
			passes += 1
		else:
			pass

	if passes == 0:
		return Number
	else:
		return False

counter = 0
for num in range(1, 100000):
	if IsPrime(num):
		counter += 1

print(f'Primes Counted: {counter}')
``` 
output:
```
Primes Counted: 9592
[Finished in 548ms]
``` 
and it calculates all the primes between 1 > 100000 (excluding 2) in about 548 milliseconds

**Comment by rbergen** [comment]
@iblissstudent I'm guessing you're aware that we already have 3 solutions in Python, all of which conform to the rules that apply to Primes solutions. Amongst others, that means that the sieve size is not 100,000 but 1,000,000.

Based on that premise, I'm not sure exactly what kind of feedback you're trying to get on this post?

**Comment by davepl** [comment]
That’s a good start!  Now make it faster, then make it faster yet.  Keep going until you can’t make it any faster!
